packages/twenty-front/src/modules/settings/data-model/fields/preview/utils/getComputedDefaultValue.ts

```diff
@@ -0,0 +1,17 @@
+// Some default value are special keywords such as 'uuid' or 'now'
+// that are mapped to a function that generates a value on the backend.
+// (see serializeFunctionDefaultValue.ts on the server)
+// We need to do a similar mapping on the frontend
+
+import { v4 } from 'uuid';
+
+export const getComputedDefaultValue = (defaultValue?: any) => {
```

Comment:
looks similar to generateEmptyFieldValue, I think we should re-use :)

Reply:
As discussed in DM I don't think it's similar